pybamm/models/full_battery_models/lithium_ion/dfn.py

```diff
@@ -32,6 +32,7 @@ class DFN(BaseModel):
 
     def __init__(self, options=None, name="Doyle-Fuller-Newman model", build=True):
         super().__init__(options, name)
+        self.x_average = False
```

Comment:
might be worth a comment here explaining why `x_average` is True/False for different models (just in dfn, spm etc., not in every submodel)